featurebyte/worker/task/batch_feature_create.py

```diff
@@ -162,15 +158,16 @@ async def is_generated_feature_consistent(
         feature = await feature_service.get_document(document_id=document.id)
         generated_hash = feature.graph.node_name_to_ref[feature.node_name]
         expected_hash = document.graph.node_name_to_ref[document.node_name]
-        is_consistent = definition == feature.definition and expected_hash == generated_hash
```

Comment:
Would it be useful to log the definitions for debugging when they don't match? Or is it expected that don't have to match?

Reply:
As the feature definition contains featurebyte version information, it triggers the inconsistency issue when the worker's featurebyte version (`document.definition`) is different from the server's featurebyte version (`feature.definition`).